pyproject.toml

```diff
@@ -1,12 +1,28 @@
 [build-system]
 requires = [
-    "setuptools >= 36.4",
+    "setuptools >= 62.5",
     "wheel",
-    "setuptools_scm >= 2.1",
+    "setuptools_scm[toml] >= 6.2",
     "cython >= 0.28.1",
 ]
 build-backend = "setuptools.build_meta"
 
+[project]
+name = "uharfbuzz"
+description = "Streamlined Cython bindings for the harfbuzz shaping engine"
+readme = "README.md"
+license = {file = "LICENSE"}
+requires-python = ">=3.5"
+authors = [
+    { name = "Adrien Tétar", email = "adri-from-59@hotmail.fr" },
```

Comment:
We need to update this as well.

Reply:
Who should be put in and in what order?

Reply:
I don’t know, HarfBuzz Team or something. Do we even need to explicitly set an author?

Reply:
Simply removing it seems to at least build and install in tox. I can remove it if everyone agrees?

Reply:
I think some author should be there. FWIW I don't see a problem keeping like it was, or if you want to add The HarfBuzz Team that's fine too.
In general, you can run `twine check` on the wheel to see if it complains about any missing metadata.

Reply:
Wouldn't you have to define that somewhere? I'll just add khaledhosny, justvanrossum and anthrotype in laphabetical order for now.